build/config/paths.js

```diff
@@ -16,7 +16,7 @@
 
 		buildDir: "build",
 		clientDir: "src",
-		clientEntryPoint: "src/toggle.js",
+		clientEntryPoint: "src/index.js",
```

Comment:
You are changing the entry point and also adding a new file `build.jakefile.js`. I hope the previous assignments work seamlessly with this new setup. Please confirm that you checked Assignments 1, 2 and 3 with this new setup.

Reply:
We added a more precise entry point for build and all previous assignments will work as expected.